packages/snowpack/src/commands/dev.ts

```diff
@@ -275,6 +236,12 @@ export async function command(commandOptions: CommandOptions) {
     await updateLockfileHash(DEV_DEPENDENCIES_DIR);
   }
 
+  // Start painting dev dashboard after successful install
```

Comment:
One annoying thing: we still have a `pino` instance in `install.ts`, which means any install errors get wiped away when the dev server starts. We don’t throw an error in `install`, currently. And we don’t allow passing a logger in that function.

What should we do here? Should we throw an error in `install.ts`? Should we add `dev` vs `build` logic in `install` to handle different logging ability? Unclear what the simplest way would be to take an install error and pass it to the dev server’s array of logs.

Reply:
@FredKSchott